src/Controller/ExportController.php

```diff
@@ -68,6 +73,9 @@ class ExportController extends AbstractController
     /** @var SalesChannelService|null */
     private $salesChannelService;
 
+    /** @var int */
+    private $total = 0;
```

Comment:
This should not be a property inside this class. Handle it as a simple variable inside the method it is actually needed and pass it, in case a method requires it.